extensions/podman/packages/extension/src/extension.ts

```diff
@@ -871,7 +880,7 @@ export async function registerProviderFor(
     },
   };
   //support edit only on MacOS as Podman WSL is nop and generates errors
-  if (extensionApi.env.isMac) {
+  if (extensionApi.env.isMac || hyperVEnabled) {
```

Comment:
What is the relation between this test and the flags above (isEditMemorySupported, isEditCPUSupported, isEditDiskSizeSupported)? If it's an OR on all of them, I think it would be better to explicitely use this OR instead, so we can change the flags without having to change here also